verilog/tools/preprocessor/verilog_preprocessor_test.sh

```diff
@@ -560,5 +569,241 @@ diff --strip-trailing-cr -u "$MY_EXPECT_FILE" "$MY_OUTPUT_FILE" || {
   exit 1
 }
 
+################################################################################
+echo "=== Test multiple-compilation-unit: including a file with an absolute path"
+
+cat > "$MY_INPUT_FILE" <<EOF
+\`include "${MY_ABSOLUTE_INCLUDED_FILE_1}"
+input_content
+EOF
+
+cat > "$MY_ABSOLUTE_INCLUDED_FILE_1" <<EOF
+included1_content
+EOF
+
+
+cat > "$MY_EXPECT_FILE" <<EOF
+${MY_INPUT_FILE}:
+(#293: "included1_content")
```

Comment:
Is the TODO to change token output to actual content output stored in an issue somewhere, so that we don't forget to change that at some point ?

Reply:
Yes, it should be the next PR.